contracts/src/LightClient.sol

```diff
@@ -426,8 +426,8 @@ contract LightClient is Initializable, OwnableUpgradeable, UUPSUpgradeable {
             revert InvalidHotShotBlockForCommitmentCheck();
         }
         for (uint256 i = stateHistoryFirstIndex; i < commitmentsHeight; i++) {
-            // The first commitment greater than the provided height is the root of the tree
-            // that leaf at that HotShot height
+            // The first HotShot commitment found that is greater than or equal to the provided
```

Comment:
I would add "which corresponding height" after "HotShot commitment found". Otherwise it seems we are comparing a commitment to some number. 

Also the last part of the sentence is hard to read: "height is the root of the tree that leaf at that HotShot height".

Reply:
yes it was confusing, updated here https://github.com/EspressoSystems/espresso-sequencer/commit/5a295bdc233406ffd908952b82bd644433111dc6